servers/republik/graphql/resolvers/MembershipStats/count.js

```diff
@@ -1,14 +1,9 @@
-const { PARKING_USER_ID } = process.env
+const moment = require('moment')
 
-module.exports = (_, args, { pgdb }) =>
-  pgdb.queryOneField(`
-    SELECT
-      count(DISTINCT m."userId")
-    FROM
-      memberships m
-    WHERE
-      ${PARKING_USER_ID ? 'm."userId" != :excludeUserId AND' : ''}
-      m.active = true
-  `, {
-    excludeUserId: PARKING_USER_ID
-  })
+const { sumBucketProps } = require('../../../lib/MembershipStats/evolution')
+
+module.exports = (_, args, context) => sumBucketProps(
+  context,
+  moment().format('YYYY-MM'),
+  { add: ['active', 'overdue'] }
+)
```

Comment:
```suggestion
) || -1
```
as `sumBucketProps` might return null

Reply:
Not falling back should throw a GraphQL error if `sumBucketProps` returns `null`, intentionally.

You sort of prefer to always return a value, even a faulty one?